src/main/java/com/github/krzysiekjodlowski/ox/Board.java

```diff
@@ -0,0 +1,110 @@
+package com.github.krzysiekjodlowski.ox;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents square game board.
+ * @author KrzysiekJodlowski
+ */
+class Board {
+    /**
+     * Size of the board game.
+     */
+    private final int boardCapacity;
+    /**
+     * Minimal boardSize.
+     */
+    private final int minBoardSize = 3;
+    /**
+     * Maximal boardSize (temporary value for now).
+     */
+    private final int maxBoardSize = 1000;
+    /**
+     * Core field in this class.
+     */
+    private final Map<FieldNumber, Symbol> moves = new HashMap<>();
+    /**
+     * Size of one side of the board.
+     */
+    private final int boardSize;
```

Comment:
Private final variables should have name contains only big letters like BOARD_CAPACITY or MIN_BOARD_SIZE

Reply:
As far as I know only final **static** variables are considered as constants and should be named uppercase;) Please correct me if I'm wrong!

Reply:
I agree with Krzysiek.